<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>办公小工具</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>482</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbOffcialTicket">
       <property name="text">
        <string>优惠机票&amp;Q</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbReimDomestic">
       <property name="text">
        <string>国内报销&amp;W</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbReimDomestic_F">
       <property name="text">
        <string>国际报销&amp;E</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pbDateCompute">
       <property name="text">
        <string>日期计算&amp;A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbWorkCalendar">
       <property name="text">
        <string>排班日历&amp;S</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbPerInfo">
       <property name="text">
        <string>人员信息&amp;D</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>60</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pbEportString">
       <property name="text">
        <string>三字码&amp;Z</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
